src/Uno.UI.Toolkit/Uno.UI.Toolkit.csproj

```diff
@@ -84,6 +84,9 @@
 		<Compile Include="$(MSBuildThisFileDirectory)..\Uno.UI\Extensions\DependencyObjectExtensions.cs">
 			<Link>Uno.UI\Extensions\%(RecursiveDir)%(FileName)%(Extension)</Link>
 		</Compile>
+	</ItemGroup>
```

Comment:
that was all that was missing!?

Reply:
That was the cause of the recent regression on ElevatedView ;-)